build-tools/installers/create-installers.targets

```
@@ -159,6 +159,7 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs" />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.debug.so')"       Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.release.so')"     Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
+    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmonodroid-mono-api.so')"       Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
```

I think you need this new file in .NET 5+ packages:
```suggestion
    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmonodroid-mono-api.so')" />
```

---

Actually list it here, and leave this line as-is:

https://github.com/xamarin/xamarin-android/blob/b987d1b6bdbef4f1e858ed2c6d2f69504f614243/build-tools/create-packs/Microsoft.Android.Runtime.proj#L39-L42

---

@jonathanpeppers I mirrored the action for `libmono-android*.so` - the two don't work without each other, so is the change you suggested also required for `libmono-android*.so`?